src/__tests__/patients/related-persons/RelatedPersons.test.tsx

```diff
@@ -18,12 +22,23 @@ const mockStore = createMockStore([thunk])
 
 describe('Related Persons Tab', () => {
   let wrapper: ReactWrapper
-
+  let history = createMemoryHistory()
+  const setup = async (location: string, patient: Patient, user: any) => {
+    history = createMemoryHistory()
+    history.push(location)
+    return mount(
+      <Router history={history}>
+        <Provider store={mockStore({ patient, user })}>
+          <RelatedPersonTab patient={patient} />
+        </Provider>
+      </Router>,
+    )
+  }
```

Comment:
Not sure if we need a setup method in this test. 

I think that doing the setup in a `beforeEach` is sufficient. Thoughts?

Reply:
If it would be more performant and reusable we make the beforeeach but I think we with the setup wrapper we can call it once on the describe scope instead of each test and in cases where we need a new patient user or route we call the setup function in the individual test. Give me a heads up and I'll finish refactoring this as necessary

Reply:
I think a setup may not be necessary because each describe has a different use case. 

I’ve found the setup function to be helpful when the same describe needs different setup.

Reply:
Reverted the test changes and added the related test!